<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectContextUi</class>
 <widget class="QDialog" name="SelectContextUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Context</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3ListView" name="contextList">
     <column>
      <property name="text">
       <string>Id</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Institute</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
     <column>
      <property name="text">
       <string>User Id</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Server</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer35">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>41</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortButton">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>contextList</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>abortButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
